### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (h1)

### ISSUE THREAD

**Issue by markreay** [ada-cat/accessibility#24]
Addie, you need to fix this right away!!!!!!

#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI))
#### Target application
[Ishinryu Karate SF](https://markreay.github.io/karatesf/initial.html)
#### Element path
#yui_3_17_2_1_1580250704041_287 > h1
#### Snippet
`<h1 id="sqs-slash-page-header" style="display: inline-block;">Where the art and the sport meet</h1>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.84 (foreground color: #999999, background color: #ffffff, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 79.0.3945.130
This accessibility issue was found using Accessibility Insights for Web 2.13.1 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
